.travis.yml

```diff
@@ -54,8 +54,8 @@ jobs:
       env: COVERAGE=yes
 
     - stage: Lint
-      install: travis_retry composer require --dev --prefer-dist phpstan/phpstan:^0.7
-      script: vendor/bin/phpstan analyse -l 3 -c phpstan.neon lib tests
+      install: travis_retry composer require --dev --prefer-dist phpstan/phpstan:^0.11 phpstan/phpstan-phpunit
+      script: vendor/bin/phpstan analyse -l 5 -c phpstan.neon lib tests
```

Comment:
Can you please move the level and paths to a phpstan.neon.dist file? Then we should be able to remove phpstan.neon and put it into the gitignore file.

Reply:
Can you also please add phpstan to the dev dependencies of composer?

Reply:
I also commited `composer.lock` as is standard in other `doctrine/*` projects.